README.md

```diff
@@ -1,2 +1,183 @@
-# Space-X
-A web application that provides commercial and scientific space travel services.
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# Space Travelers' Hub
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Space Travellers' Hub <a name="about-project"></a>
+
+**Space Travellers' Hub** is a React-based single-page application (SPA) that uses real-time data from the SpaceX API. Users can book rockets and participate in selected space missions, and the services they choose are advertised on their profile page.
+
+The project is built with various programming tools such as React, JSX, CSS, and JavaScript. It is also deployed on Netifly, so you can try it out by visiting the live demo.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- React
+- JSX
+- CSS
+- Javascript ES6
+- Visual Studio Code
+- Jest
+- React Testing Library
+- ESLint
+- Stylelint
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> The three key features of this project include the following.
+
+- **Book SpaceX rockets and missions, cancel the previously made booking.**
+- **Join SpaceX space missions, leave the mission the user he/she has joined earlier.**
+- **View all reserved rockets, space missions, and dragons.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [click for live demo](https://xghayor.github.io/Space-X/)
```

Comment:
- [ Optional ] I think it would be nice if you fixed the live demo link on your readme file or the deployment of your application. Currently, when I click on the live demo link it leads me to a blank white page. It would be nice and professional if the deployed version of your App works the same online as the local version does. Please see the screenshot below and rectify this.

## Live deployment
![Screenshot from 2023-09-07 14-45-26](https://github.com/Xghayor/Space-X/assets/16392046/60f6047b-a2bc-4bb5-bdeb-1d79a830309e)